x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_process_detail.tsx

```diff
@@ -1,187 +1,187 @@
-          <CubeForProcess
-            isProcessTerminated={isProcessTerminated}
-            isProcessOrigin={isProcessOrigin}
-          />
-          {processName}
-        </h4>
-      </EuiTitle>
-      <EuiText>
-        <EuiTextColor color="subdued">
-          <span id={titleId}>{descriptionText}</span>
-        </EuiTextColor>
-      </EuiText>
-      <EuiSpacer size="l" />
-      <StyledDescriptionList
-        type="column"
-        align="left"
-        titleProps={{ className: 'desc-title' }}
-        compressed
-        listItems={processInfoEntry}
-      />
-    </>
-  );
-});
-ProcessDetails.displayName = 'ProcessDetails';
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { memo, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  htmlIdGenerator,
+  EuiSpacer,
+  EuiTitle,
+  EuiText,
+  EuiTextColor,
+  EuiDescriptionList,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+import * as event from '../../../../common/endpoint/models/event';
+import { CrumbInfo, formatDate, StyledBreadcrumbs } from './panel_content_utilities';
+import {
+  processPath,
+  processPid,
+  userInfoForProcess,
+  processParentPid,
+  md5HashForProcess,
+  argsForProcess,
+} from '../../models/process_event';
+import { CubeForProcess } from './process_cube_icon';
+import { ResolverEvent } from '../../../../common/endpoint/types';
+import { useResolverTheme } from '../assets';
+
+const StyledDescriptionList = styled(EuiDescriptionList)`
+  &.euiDescriptionList.euiDescriptionList--column dt.euiDescriptionList__title.desc-title {
+    max-width: 10em;
+  }
+`;
+
+/**
+ * A description list view of all the Metadata that goes with a particular process event, like:
+ * Created, Pid, User/Domain, etc.
+ */
+export const ProcessDetails = memo(function ProcessDetails({
+  processEvent,
+  isProcessTerminated,
+  isProcessOrigin,
+  pushToQueryParams,
+}: {
+  processEvent: ResolverEvent;
+  isProcessTerminated: boolean;
+  isProcessOrigin: boolean;
+  pushToQueryParams: (queryStringKeyValuePair: CrumbInfo) => unknown;
+}) {
+  const processName = event.eventName(processEvent);
+  const processInfoEntry = useMemo(() => {
+    const eventTime = event.eventTimestamp(processEvent);
+    const dateTime = eventTime ? formatDate(eventTime) : '';
+
+    const createdEntry = {
+      title: '@timestamp',
+      description: dateTime,
+    };
+
+    const pathEntry = {
+      title: 'process.executable',
+      description: processPath(processEvent),
+    };
+
+    const pidEntry = {
+      title: 'process.pid',
+      description: processPid(processEvent),
+    };
+
+    const userEntry = {
+      title: 'user.name',
+      description: userInfoForProcess(processEvent)?.name,
```

Comment:
for this file, try disabling whitespace: 
![image](https://user-images.githubusercontent.com/35559/88582198-50b62f00-d01c-11ea-8dd2-f10133c37f41.png)
My editor switched it to unix line endings.